vb-fe/src/_store/system/SystemReducer.ts

```diff
@@ -10,9 +11,9 @@ export interface SystemState {
   error: any;
   isAuthenticated: boolean;
   /**
-   * Denotes if the create room modal is open
+   * Denotes if the room modal is open
    */
-  createRoomOpen: boolean;
+  roomModal: RoomModalType;
```

Comment:
shouldn't this be `RoomModalType | null` as you're initializing to `null` at the bottom